### ISSUE TITLE
Help getting this to work

### ISSUE THREAD

**Issue by avioli** [mheap/trello-cli#72]
Hi there. As a terminal user I love cli tools of all services I start using.

I'm trying to get this running, but I reached a point early on and am stuck. Any help will be appreciated.

- I've cloned the repo
- ran `npm install`
- ran `./bin/trello`
- visited the link and copied the api key, then replaced YOURAPIKEY in `$HOME/.trello-cli/config.json`
- ran `./bin/trello`
- visited the generated link and copied the token
- ran `./bin/trello set-auth "$TOKEN"`
- ran `./bin/trello --help`
- ran `./bin/trello show-boards` and got:

```
(node:29020) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
```

I'm running node.js 14.12.0 and am on macOS Mojave 10.14.6.

Any help?

**Comment by avioli** [comment]
Just ran `./bin/trello refresh` and it logged same above issue, but did some work (I guess refreshed its data), so now `./bin/trello show-boards` outputs the boards with their ids.

Running `node --trace-warnings ./bin/trello show-boards` outputs the boards with their ids and then:

```
(node:29483) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
    at emitCircularRequireWarning (internal/modules/cjs/loader.js:668:11)
    at Object.get (internal/modules/cjs/loader.js:682:5)
    at Object.exports.setLevels (/Users/avioli/repos/trello-cli/node_modules/winston/lib/winston/common.js:35:14)
    at Object.<anonymous> (/Users/avioli/repos/trello-cli/node_modules/winston/lib/winston.js:84:8)
    at Module._compile (internal/modules/cjs/loader.js:1076:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:941:32)
    at Function.Module._load (internal/modules/cjs/loader.js:782:14)
    at Module.require (internal/modules/cjs/loader.js:965:19)
    at require (internal/modules/cjs/helpers.js:88:18)
```

**Comment by avioli** [comment]
I guess #62 is the same issue, but I'm getting some node.js warning instead of nothing.

**Comment by igor-krupenja** [comment]
Also getting this warning on every trello-cli command though it seems to be a cosmetic issue only: boards are showing, cards can be added successfully, etc.

**Comment by mheap** [comment]
Yep @igor-krupenja, it was a cosmetic issue. I've just released `v0.4.0` with a fix.

@avioli Are things now working for you once you've run `trello refresh`?

**Comment by mheap** [comment]
Closing due to lack of activity
